src/SDL12_compat.c

```diff
@@ -2401,8 +2401,34 @@ EventFilter20to12(void *data, SDL_Event *event20)
             PendingKeydownEvent.key.keysym.unicode = 0;
 
             /* If Unicode is not enabled, flush all KEYDOWN events immediately. */
-            if (!EnabledUnicode)
+            if (!EnabledUnicode) {
                 FlushPendingKeydownEvent(0);
+                return 1;
+            }
+
+            /* some programs rely on unicode values for these control characters */
+            switch (PendingKeydownEvent.key.keysym.sym)
+            {
+                case SDLK12_BACKSPACE:
+                    FlushPendingKeydownEvent('\b');
+                    break;
+                case SDLK12_TAB:
+                    FlushPendingKeydownEvent('\t');
```

Comment:
Do we need a break statement here?

Reply:
Whoops! Yup, thanks for catching that. Uploading a fixed version now…